concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide Columns.md

```diff
@@ -0,0 +1,202 @@
+When the total width of columns exceeds the component's width, the TreeList truncates column cell texts or adds horizontal scrolling. As an alternative, the component can hide one or several columns to prevent horizontal scrolling and display cell texts in full. To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
```

Comment:
```suggestion
When the total width of columns exceeds component width, the TreeList either truncates column cell text or adds horizontal scrolling. As an alternative, the component can hide one or several columns to prevent horizontal scrolling and display cell text in full. To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
```